src/main/scala/nl/knaw/dans/bag/Deposit.scala

```diff
@@ -336,6 +336,11 @@ class Deposit private(val baseDir: File,
     withDepositProperties(newProperties)
   }
 
+  /**
+   * Most methods in this library only manipulate the Bag and Deposit in memory.
+   * Saves the in-memory deposit to the file system, like calling flush() in a Stream. It writes all the bag-files and the deposit.properties.
```

Comment:
1. There is no `flush()` on a [`Stream`](https://www.scala-lang.org/api/current/scala/collection/immutable/Stream.html). Perhaps remove the whole _like calling flush() in a Stream_ all together. It doesn't add much here.
2. Refer to the `bag.save()` and `props.save` functions (in the text and maybe also using `@see`.
3. Don't forget to fill in the `@return` line.